zombienet_tests/functional/0001-basic-network.toml

```diff
@@ -0,0 +1,22 @@
+[relaychain]
+chain_spec_path = "chain/westend-local/westend-local-spec-raw.json"
+
+chain = "westend-local"
+
+[[relaychain.nodes]]
+name = "alice"
+command = "gossamer"
+validator = true
+args = ["--key alice", "--babe-lead"]
```

Comment:
Imagine that this flag will need to be removed with @dimartiro work right?

Reply:
Yep, I tested the zombienet without this flag using my changes and I worked and I just merged.
@edwardmack could you rebase develop and remove this flag please?